src/runtime/node/vm/index.ts

```diff
@@ -0,0 +1,42 @@
+import noop from "../../mock/noop";
+import type vm from "node:vm";
+import { Script } from "./script";
+import * as constants from "./constants";
+
+export { Script } from "./script";
+export * as constants from "./constants";
+
+export const compileFunction: typeof vm.compileFunction =
+  function compileFunction() {
+    return noop;
```

Comment:
I think that the returned value should be a function that throws when called, or compileFunction itself should throw when called.